README.md

```diff
@@ -56,4 +57,7 @@ npx lerna create '@chanzuckerberg/czedi-kit-<package-name>' \
 
 ### Publishing
 
-We're not ready to publish yet
+**We are only publishing alpha pre-releases at this time**
+
+1. Confirm everything is green on CI
+2. Run `lerna publish ???`
```

Comment:
what andrew suggested:
```
lerna publish --canary --preid alpha
```

what I actually did:
- find + replace all version numbers
- update CHANGELOGs
- manually create a tag + release
- panic???
- run `lerna publish from-package`

@dcwither as lerna expert, what do you recommend / what have you been doing?

Reply:
"expert" lol.

```
lerna publish --canary --preid alpha
```

Should work, though it won't give a commit message that agrees with our CI/commit linter.

What I've been doing to make sure my publishes will contain what I expect:

```bash
git checkout master 
npx lerna version --no-push --conventional-commits --cd-version prerelease --message "chore(release): publish v0.0.1-alpha.<version>"
# confirm tags/commit/changelog etc
npx lerna publish from-git --no-push
git push origin --tags
git push origin master # not sure if this step is necessary
```

This is also why it would be great to get a publish script out.

The `--no-push` means that I don't have to delete remote stuff after a messed up publish.

Reply:
oof okay thanks! will add all that + context